src/lib/statistics/attribute_statistics.cpp

```diff
@@ -92,6 +99,10 @@ std::shared_ptr<BaseAttributeStatistics> AttributeStatistics<T>::sliced(
     }
   }
 
+  if (distinct_value_count) {
```

Comment:
Can you please check if the `clang-tidy is crazy` part is still necessary above?

Reply:
Could not reproduce it with clang-14/macOS. Removed the `NOLINT`s.